choco/VERIFICATION.txt

```diff
@@ -0,0 +1,7 @@
+VERIFICATION
+Verification is intended to assist the Chocolatey moderators and community
+in verifying that this package's contents are trustworthy.
+
+This package is published by the NUnit Project itself. The binaries are 
+identical to other package types published by the project, in particular
+the NUnit.ConsoleRunner and NUNit.Console nuget packages.
```

Comment:
I'll try to reword and fix the spelling. I was trying to avoid having two files for the two packages.